Add files via upload

GeorgDiener opened the pull request:
Hello, I added a Salad with baked goat cheese.

GeorgDiener committed:
Add files via upload

GeorgDiener committed:
Update config.js

jlooper commented on src/recipes/Healthy-salad-with-baked-goat-cheese.md:
```diff
@@ -0,0 +1,55 @@
+---
+tags: ["Salad", "Cheese", "Peach"]
```
hi, can you just have this as one of the categories - Salad is perfect, so remove 'cheese' and 'peach' please.

jlooper commented on src/recipes/Healthy-salad-with-baked-goat-cheese.md:
```diff
@@ -0,0 +1,55 @@
+
+Cut the peaches into triangels and put them in the pan with a table spoon of olive oil. Frie them on medium heat for 10 minutes.   
+
+While frying, cut the goat chees into round pieces und place them on a oven tray. Put the honey over them and then crunsh the wallnuts over the honney-soaked cheese. Bake the cheese for 5 minutes at 180 degrees. 
```
little typo (cheese) and Frie should be 'fry'

jlooper replied:
also typo : crush and honey

jlooper commented on src/recipes/Healthy-salad-with-baked-goat-cheese.md:
```diff
@@ -0,0 +1,55 @@
+
+While frying, cut the goat chees into round pieces und place them on a oven tray. Put the honey over them and then crunsh the wallnuts over the honney-soaked cheese. Bake the cheese for 5 minutes at 180 degrees. 
+
+While backing, wash the salad and place it on two plates. 
```
baking

jlooper commented on src/recipes/Healthy-salad-with-baked-goat-cheese.md:
```diff
@@ -0,0 +1,55 @@
+
+While backing, wash the salad and place it on two plates. 
+
+For the dressing: Mixe 3 spoones of olive oil with the balsamic vinegar. Then add the sweet mustard and the honey. Give it some more flavour by adding salt and pepper (try to tast it inbetween). Mix it all. 
```
spoons

taste

in between

jlooper commented on src/recipes/Healthy-salad-with-baked-goat-cheese.md:
```diff
@@ -0,0 +1,55 @@
+
+Get the peaches out of the pan and place them on the two salad plates. 
+
+As the last step: get the freshly baked cheese out of the oven and put them ontop wthe salad. 
```
on top of the salad

jlooper commented on src/recipes/Healthy-salad-with-baked-goat-cheese.md:
```diff
@@ -0,0 +1,55 @@
+
+## Tips
+
+Add some extra flavour with some cranberrys.
```
cranberries

jlooper reviewed (changes requested):
hi! if you can fix the tags and typos I'm happy to add this. looks good!

jlooper reviewed (commented):
Thanks for the recipe — the Healthy Salad with Baked Goat Cheese content is great! 🌿🧀

This PR was opened back in **Oct 2022**, and the repo has since been migrated from VuePress to VitePress (commit `b4dce0a`, Jan 2025). Because of that, this PR no longer merges cleanly and needs a small update. I verified all of this by testing locally:

**Issues found:**
1. **`src/.vuepress/config.js`** — this file was **deleted** during the VitePress migration, so this change can't apply (git reports a modify/delete conflict). The new site config lives in `.vitepress/config.mts`, where recipes are registered in a hand-curated sidebar.
2. **`src/recipes/Healthy-salad-with-baked-goat-cheese.md`** — recipes moved from `src/recipes/` to `recipes/` at the repo root (with lowercase kebab-case filenames), so this file lands in the wrong directory.

**What works (verified with a local build):**
I ported the recipe to the current layout as `recipes/healthy-salad-with-baked-goat-cheese.md`, added it to the **Soups & Salads** sidebar group in `.vitepress/config.mts`, and ran `npm run build` — it passes and the page renders with its `<TagLinks />` tags. ✅

**To bring this PR up to date:**
- Move the file: `git mv src/recipes/Healthy-salad-with-baked-goat-cheese.md recipes/healthy-salad-with-baked-goat-cheese.md`
- Drop the `src/.vuepress/config.js` change
- Add a sidebar entry in `.vitepress/config.mts`, e.g. under *Soups & Salads*:
  ```js
  { text: 'Healthy Salad with Baked Goat Cheese', link: '/recipes/healthy-salad-with-baked-goat-cheese' },
  ```

Nice-to-have: the recipe text has a few small typos ("spoones", "Frie", "chees", "cranberrys") worth a quick proofread. Happy to push these updates to your PR branch if you'd like!

jlooper committed:
Fix typos and improve clarity in salad recipe

Corrected spelling and grammatical errors in the recipe, including ingredient measurements and cooking instructions.

jlooper committed:
Delete src/.vuepress/config.js
